zcash_client_sqlite/src/wallet/init/migrations/addresses_table.rs

```diff
@@ -0,0 +1,176 @@
+use std::collections::HashSet;
+
+use rusqlite::{Transaction, NO_PARAMS};
+use schemer::Migration;
+use schemer_rusqlite::RusqliteMigration;
+use uuid::Uuid;
+use zcash_client_backend::{address::RecipientAddress, keys::UnifiedFullViewingKey};
+use zcash_primitives::{consensus, zip32::AccountId};
+
+use super::super::{add_account_internal, WalletMigrationError};
+
+#[cfg(feature = "transparent-inputs")]
+use zcash_primitives::legacy::keys::IncomingViewingKey;
+
+/// The migration that removed the address columns from the `accounts` table, and created
+/// the `accounts` table.
+///
+/// d956978c-9c87-4d6e-815d-fb8f088d094c
+pub(super) const ADDRESSES_TABLE_MIGRATION: Uuid = Uuid::from_fields(
```

Comment:
This is a nice approach, I'll have to handle the ID for the views migration the same way.

Reply:
Yep; I figured this way we could use code imports for DAG edges.